include/aotriton/config.h.in

```diff
@@ -19,6 +19,12 @@
 #cmakedefine AOTRITON_VERSION_PATCH @AOTRITON_VERSION_PATCH@
 #cmakedefine AOTRITON_GIT_SHA1 @AOTRITON_GIT_SHA1@
 
+#include <aotriton/export.h>
+
+#ifdef _MSC_VER
```

Comment:
Duplicated

If you decided to use cmake's GenerateExportHeader, then you don't need a separate macro to check the compiler.